tests/compiled_templates/bird.cfg

```diff
@@ -0,0 +1,99 @@
+# Generated by confd
```

Comment:
These are the expected outputs right? Maybe the directory should be called expected outputs or something.

What's the plan for when these change? Could you include some comments, or documentation guiding how to fix these tests up when they start failing becuase we changed the templates in a different repo.